client/lib/plans/personal-plan.js

```diff
@@ -54,13 +81,18 @@ export const insertSitePersonalPlan = plans => {
 		has_domain_credit,
 		product_name,
 		product_slug,
-		raw_price
-	} = personalPlan;
+		raw_price,
+		currency_code
+	} = formatPlan( plans );
 
 	if ( ! ( isEmpty( plans ) || has( plans, product_id ) ) ) {
+		const hasCurrentPlan = some( plans, matchesProperty( 'current_plan', true ) );
```

Comment:
Not sure if it's worth it here, but we could turn this dynamic closure into a static function.

```js
const hasCurrentPlan = partialRight( some, matchesProperty( 'current_plan', true ) );
```

Now it's point-free (no variables, so pretty) _and_ we can define it out in the module scope instead of inside this function.

```js
current_plan: ! hasCurrentPlan( plans )
```